lib/value/src/value/convert.rs

```diff
@@ -264,6 +264,12 @@ impl<const N: usize> From<&[u8; N]> for Value {
     }
 }
 
+impl From<&[u8]> for Value {
+    fn from(data: &[u8]) -> Self {
+        Self::from(Bytes::copy_from_slice(data))
+    }
+}
+
```

Comment:
I added this to fit the type signature of the code, but in general I think I have a lot to learn about the Vector types. Would appreciate feedback on whether this is necessary or if there's a better way to use the existing trait implementations, given the code in the `chunks` method